### ISSUE TITLE
Bunyan is quite chatty, offer a silence feature

### ISSUE THREAD

**Issue by jeffwilcox** [AzureAD/passport-azure-ad#72]
I have not used Bunyan in the past, but since integrating this great library into my app for AAD auth, I've been inundated with the debug output it creates.

Ideally a configuration option to "silence" the Bunyan output would be nice, or a different logging system which would be a little less chatty for those using this in their apps.

**Comment by brandwe** [comment]
Reading up I've noticed you can launch it only for warn like: 

`$ node hi.js | bunyan -l warn`

That should reduce the verboseness substantially. Sadly if you don't use bunyan pipe  at all you get the full dump in json to the terminal. I think the best approach is to change this to write to a log instead of stdout. 

```
 {
      level: 'error',
      path: '/var/tmp/myapp-error.log'  // log ERROR and above to a file
    }

```

Which will be my next approach if I don't find a better logger.

**Comment by jeffwilcox** [comment]
Got it. Well, if that's a potential way to make it work, a way to pass in a log location (or log configuration) in the strategy options would be great. That way I could just log to `/dev/null`... and with `error` level at least tokens and other critical data would not be located in the logs.

**Comment by brandwe** [comment]
Fixed in #82.
